src/lib/helper.js

```diff
@@ -27,10 +27,22 @@ const getUTCNowInMilliSec = () => {
   const offset = now.getTimezoneOffset() * 60000;
   return time + offset;
 };
+const hourInMilliSeconds = 1000 * 60 * 60;
+const dayInMilliSeconds = hourInMilliSeconds * 24;
 
-const fromMilliSecToHours = (time) => time / (1000 * 60 * 60);
+const fromMilliSecToHours = (time) => time / hourInMilliSeconds;
 
-const fromMilliSecToDays = (time) => Math.ceil(time / (1000 * 60 * 60 * 24));
+const fromMilliSecToDays = (time) => Math.ceil(time / dayInMilliSeconds);
+
+//To check if the item is out of date, subtract the recentPurchase (in days) from the current date (in days)
+//and compare if the result is greater than the estimated howSoon X 2.
+// we're also checking if recentPurchase because when items are first added to the list
+// recentPurchase is null by default. And items are not out of date if they're just added.
+export const isOutOfDate = (howSoon, recentPurchase) =>
+  recentPurchase &&
+  fromMilliSecToDays(getUTCNowInMilliSec()) -
+    fromMilliSecToDays(recentPurchase) >
+    howSoon * 2;
```

Comment:
This looks fine. But one thing I'm not sure is that when I add a new `item` with the following parameters, the item is categorized as inactive:

`howSoon` = 7,
`recentPurchase` = as yesterday ( in milliseconds: 1605284587 ),
`numberOfPurchases` = 1 

Shouldn't the item be active when the elapsed time since the last purchase is less than 2x estimated? 
This could easily be my issue while testing, but would be great if you could confirm it.

Reply:
Hey, thank you so much for the review. :))

I think the time you added was in seconds, not in milliseconds. If you'd like to check it again please use [this converter](https://datetomilliseconds.netlify.app/) to change the dates to milliseconds. I was adding it to the PR but then realized that it wasn't working well but it's fixed now. :) 

Working with these all milliseconds this week hurt my eyes, to be honest. 😄

Reply:
ah that's right. Thanks for pointing that out. 
Did you build this converter?  👀👌🏻

Just double-checked and everything works perfectly.
You did a fantastic job!!! 👏

Reply:
Yes. It was hard to test things without it. I added it to the PR but then I realized that I missed checking for timezones so I had to delete it but Sarah pointed out that I should've added an offset to fix the timezone bug. :)

Thank you. :)) and thank you so much for your time. 🌹